transyto/noise/__init__.py

```diff
@@ -66,7 +66,7 @@ def compute_noises(gain, exptime, target_flux_sec, target_background_sec,
                                                          * gain * exptime)) / target_signal)
 
     # Sigma readout noise
-    readout_noise = (pixel_readout * aperture_radius)**2 * np.pi
+    readout_noise = (pixel_readout) * (aperture_radius)**2 * np.pi
     sigma_readout = -2.5 * np.log10((target_signal - np.sqrt(readout_noise)) / target_signal)
```

Comment:
If you check in the CCD equations (just Google, or in my thesis), the `readout_noise` goes squared. However, in this line (i.e. 70) it needs to have the same units of the `target_signal` (i.e. e-), so initially it needs to be squared so when you take the `sqrt()` you don't end up with units of e-^0.5.

Reply:
And the log function is because, we are converting the flux to the magnitude scale?